app/views/incomeSources/add/ForeignPropertyAddedObligations.scala.html

```diff
@@ -0,0 +1,178 @@
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import views.html.layouts.unifiedLayout
+@import views.html.components._
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import uk.gov.hmrc.govukfrontend.views.html.components.FormWithCSRF
+@import views.html.helpers.injected.ErrorSummary
+@import models.incomeSourceDetails.viewmodels.ObligationsViewModel
+@import java.time.LocalDate
+@import _root_.implicits.ImplicitDateFormatterImpl
+
+
+@this(
+  mainTemplate: unifiedLayout,
+  h1: h1,
+  implicitDateFormatter: ImplicitDateFormatterImpl,
+  continueButton: ContinueButton,
+  formWithCSRF: FormWithCSRF,
+  errorSummary: ErrorSummary,
+  govukTable: GovukTable,
+)
+
+@(sources: ObligationsViewModel, postAction: Call, backUrl: String, isAgent: Boolean)(implicit messages: Messages, user: auth.MtdItUser[_])
+
+@getMessage(key: String, args: String*) = @{
+  messages(s"business-added.$key", args: _*)
+}
+
+@import implicitDateFormatter.longDate
+
+@mainTemplate(pageTitle = getMessage("foreign-property-h1") + " " + getMessage("h2"),
+  backUrl = Some(backUrl),
+  isAgent = isAgent,
+  btaNavPartial = user.btaNavPartial) {
+
+    @formWithCSRF(action = postAction) {
+
+        <div class="govuk-panel govuk-panel--confirmation" id="banner">
+            <h2 class="govuk-panel__body govuk-!-margin-bottom-3">
+                @getMessage("foreign-property-head")
+            </h2>
+            <h1 class="govuk-panel__title">
+                @getMessage("foreign-property-h1")
+            </h1>
+            <h2 class="govuk-panel__body">
+                @getMessage("foreign-property-base")
+            </h2>
+        </div>
+
+        <h2 class="govuk-label-wrapper">
+            <label class="govuk-label govuk-label--l govuk-!-margin-top-8">
+                @getMessage("text")
+            </label>
+        </h2>
+
+        @if(sources.quarterlyObligationsDatesYearOne.nonEmpty){
+            <div class="box-simple" id="quarterly">
+                <div class="page_headers govuk-heading-m"> @getMessage("quarterly-heading") </div>
+                <p class="govuk-body"> @getMessage("quarterly-p1") </p>
+
+                <table class="govuk-table">
+                    <thead class="govuk-table__head">
+                        <tr class="govuk-table__row">
+                            <th scope="col" class="govuk-table__header govuk">@(getMessage("table.heading-1")+" "+sources.quarterlyObligationsDatesYearOne.head.inboundCorrespondenceFrom.getYear+" "+getMessage("to")+" "+(sources.quarterlyObligationsDatesYearOne.head.inboundCorrespondenceFrom.getYear + 1))</th>
```

Comment:
.getYear returning LocalDate, and we need apply formatting for LocalDate type, like toLongDate;

Reply:
as discussed Long date not needed here